promql/promqltest/testdata/functions.test

```diff
@@ -749,7 +742,7 @@ eval instant at 1m avg_over_time(metric8[1m])
 
 # This overflows float64.
 eval instant at 1m sum_over_time(metric8[1m])/count_over_time(metric8[1m])
```

Comment:
Maybe, in this case, do a `1m1s` range or something to still provoke the overflow.

Reply:
`metric8` is declared as:
```
load 10s
  metric8 9.988465674311579e+307 9.988465674311579e+307
```
So `eval instant at 1m count_over_time(metric8[1m1ms])` shall be 2.

Reply:
We have a misunderstanding here. The purpose of this test is to create a `Inf` result. so the comment above should stay, and the test expression should be `eval instant at 1m sum_over_time(metric8[2m])/count_over_time(metric8[2m])` (or similar).

Reply:
Get it. Resolved.